### PR TITLE
Update readme, remove Using section and add links

### PR DESCRIPTION
**Pull request by tetchel**
Signed-off-by: Tim Etchells <timetchells@ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,23 +25,23 @@ Codewind for VS Code provides tools to help build cloud-native, containerized ap
...
+2. Open the **Codewind** view in the Explorer view group or enter `Focus on Codewind View` into the [**Command Palette**](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette). If you do not see Codewind in either the Explorer view or the Command Palette, the extension did not install correctly.
 3. Choose **Install** when prompted to complete the installation of additional Docker images that Codewind requires to run. The installation may take a few minutes to complete.
-
-## Using Codewind for VS Code
-To see the actions available, open the [**Command Palette**](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) and type `Codewind`.</br>
+4. Start by creating or adding a project. You can add a project by using either  the `Create Project` or `Add Existing Project` command. The new project appears in the Codewind view.
```
Comment (sishida) [inline README.md]: - I understand what you mean, but it still sounds unusual to me to say "Start" on something that's step 4. Maybe instead say something like, "Now you can create or add a project. Add a project by..."
- Remove extra space after "either."

File: README.md [hunk 2]
```diff
@@ -23,23 +25,23 @@ Codewind for VS Code provides tools to help build cloud-native, containerized ap
...
+5. To see all actions available, open the Command Palette and type `Codewind`. Commands can also be accessed through buttons and right-click menus in the Codewind view.
```
Comment (sishida) [inline README.md]: Remove passive voice in the second sentence by changing it to "You can also access commands through..."

File: README.md [hunk 3]
```diff
@@ -23,23 +25,23 @@ Codewind for VS Code provides tools to help build cloud-native, containerized ap
...
+2. Open the **Codewind** view in the Explorer view group or enter `Focus on Codewind View` into the [**Command Palette**](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette). If you do not see Codewind in either the Explorer view or the Command Palette, the extension did not install correctly.
 3. Choose **Install** when prompted to complete the installation of additional Docker images that Codewind requires to run. The installation may take a few minutes to complete.
-
-## Using Codewind for VS Code
-To see the actions available, open the [**Command Palette**](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) and type `Codewind`.</br>
+4. Now you can create or add a project. Add a project by using either the `Create Project` or `Add Existing Project` command. The new project appears in the Codewind view.
+5. To see all actions available, open the Command Palette and type `Codewind`. You can also access Commands through buttons and right-click menus in the Codewind view.
```
Comment (sishida) [inline README.md]: I think "commands" is generic enough in this context to be lower case. Also...should we be more specific about which buttons and which menus that users should go to so they can access the commands?

Comment (tetchel) [reply README.md]: I don't want to here because it is pretty complex, let me add a link to that doc page

**Pull request merged**